script.js

```diff
@@ -0,0 +1,18 @@
+const hamburger = document.querySelector('.fa-bars');
+const MBMenu = document.querySelector('.mb_menu');
+const cross = document.querySelector('#x_icon');
+const menuItems = document.getElementsByClassName('menu_list_items');
+
+hamburger.addEventListener('click', () => {
+  MBMenu.style.display = 'block';
+});
+
+cross.addEventListener('click', () => {
+  MBMenu.style.display = 'none';
+});
+
+for (let i = 0; i < menuItems.length; i += 1) {
+  menuItems[i].addEventListener('click', () => {
+    MBMenu.style.display = 'none';
+  });
+}
```

Comment:
- [ ] As [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) for this project, your project is still missing the following requirements.

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

Please kindly implement the above changes